src/contractTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/consumer/ccd/CcdCasePactTest.java

```diff
@@ -0,0 +1,279 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi.consumer.ccd;
```

Comment:
I think we should split this into a single test class per PACT. It will be easily readable, maintainable and will also focus responsibility on per test.

We should use that approach and standard across our tests.